src/Misc/externals.sh

```diff
@@ -151,36 +154,46 @@ if [[ "$PACKAGERUNTIME" == "win-arm64" ]]; then
     # todo: replace these with official release when available
     acquireExternalTool "$UNOFFICIAL_NODE_URL/v${NODE16_VERSION}/$PACKAGERUNTIME/node.exe" node16/bin
     acquireExternalTool "$UNOFFICIAL_NODE_URL/v${NODE16_VERSION}/$PACKAGERUNTIME/node.lib" node16/bin
+    acquireExternalTool "$UNOFFICIAL_NODE_URL/v${NODE18_VERSION}/$PACKAGERUNTIME/node.exe" node18/bin
+    acquireExternalTool "$UNOFFICIAL_NODE_URL/v${NODE18_VERSION}/$PACKAGERUNTIME/node.lib" node18/bin
```

Comment:
NOTE: They do NOT yet exist. For 16, we can avoid the problem by keeping the version.
Consult here: https://github.com/nodejs/unofficial-builds/issues/71

Reply:
Published!